bin/typescript-axios-petstore-all.sh

```diff
@@ -3,5 +3,6 @@
 ./bin/typescript-axios-petstore-target-es6.sh
 ./bin/typescript-axios-petstore-with-npm-version.sh
 ./bin/typescript-axios-petstore-with-npm-version-and-separate-models-and-api.sh
+./bin/typescript-axios-petstore-with-complex-headers.sh
```

Comment:
please add this to the windows scripts in `bin/windows/typescript-axios-petstore-all.bat`

Reply:
done